boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -50,6 +50,22 @@
 			};
 		};
 
+		etr_buf_ram0x_region: memory@2f0be000 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2f0be000 DT_SIZE_K(4)>;
+			status = "disabled";
+			perm-read;
+			perm-write;
+			perm-secure;
```

Comment:
Considering this node is `perm-secure`, it should be a sub-node of `cpuapp_ram0x_region`, neighboring `cpuapp_data`. We cannot afford multiple regions with the same permissions.

Reply:
I add this configuration so it is aligned with the custom SDFW I prepared for testing. It needs to be aligned when SDFW with STM support via UICR is settled.

Reply:
My point is that we have limited number of MPC OVERRIDEs. So when two memory partitions have the same properties, they should be covered by a single `owned-memory` node.

I don't fully follow how custom secdomfw build depends on this entry. If the custom build includes some hardcoded addresses it should be fine to customize them to match this dts file.

Reply:
Anyway this part needs to be aligned to SDFW so lets wait for that.

Reply:
The secure domain firmware needs to know the etr buffer location at build time. So we are flexible where to place it right now but it will need to be 'frozen' in the future once the sdfw is finalized. There isn't a spot in the uicr to report an etr location as of now.

Reply:
I thought we decided in an offline discussion we are going to create a new compatible value to describe this buffer being a subnode of `cpuapp_ram0x_region`.

Is this a temporary solution until the binding is present? If yes, how about a TODO comment so that our intention would be clear to whoever reads the code.

Reply:
Also, it looks this new binding must be cache line aligned and the size must be multiplication of cache lines. It should be documented unless we can somehow enforce it by binding description.

Reply:
@nordic-krch , what is the plan around this device tree node? Do you intend to keep it as is temporary?

Reply:
I've just noticed you commented in https://github.com/zephyrproject-rtos/zephyr/pull/72476#issuecomment-2160239001